tests/plugins/autolink/autolink.js

```diff
@@ -134,6 +143,29 @@
 			}
 		},
 
+		// (#3156)
+		'test various valid URL links with optional regex': function() {
```

Comment:
Unfortunately, all unit tests (not only this one) seem to be falsely positive due to similarity with default regex. It can be easily verified by checking out `autolink` plugin file into the upstream branch. E.g. `git checkout major -- plugins/autolink/plugin.js` and running bender tests. Tests should pass only on PR branch. Simplifying regexes should fix the issue.